server/controllers/messages/addChannel.js

```diff
@@ -1,3 +1,10 @@
-exports.addChannel = (req, res) => {
-  res.send('Hi from posting channels');
+const { messages: { addChannel, addSubscriber } } = require('../../database/queries');
+
+exports.addChannel = (req, res, next) => {
+  const { channelName, subscribers } = req.body;
+  addChannel(channelName)
```

Comment:
where is the validation :open_mouth:

Reply:
I did it ^_^
But I have a question 
Are we need to ensure that channel name is not already existing or will do that in the front end ??
@Fatmasiam

Reply:
@MaiUbeid 
We don't need to ensure if it's exists 
The channel name can repeat or the same for two-channel or more